sim/druid/lacerate.go

```diff
@@ -44,13 +50,13 @@ func (druid *Druid) registerLacerateSpell() {
 
 		ApplyEffects: core.ApplyEffectFuncDirectDamage(core.SpellEffect{
 			ProcMask:         core.ProcMaskMeleeMHSpecial,
-			DamageMultiplier: 1,
+			DamageMultiplier: lbdm * dwdm,
 			ThreatMultiplier: 0.5,
 			FlatThreatBonus:  267,
 
 			BaseDamage: core.BaseDamageConfig{
 				Calculator: func(sim *core.Simulation, hitEffect *core.SpellEffect, spell *core.Spell) float64 {
-					damage := 88 + 0.01*hitEffect.MeleeAttackPower(spell.Unit)
+					damage := 88 + initialBonus + 0.01*hitEffect.MeleeAttackPower(spell.Unit)
```

Comment:
This wont include attack power bonuses on the target. You can check the dk spells or  or the base attack power calculator for how to include them

Reply:
sure, updated the dot tick / initial damage

Reply:
You are not multiplying the bonus ap with the 0.1 modifier as multiplication is done before addition

Reply:
If feral uses custom ap damage formulas i suggest you wrap that into a helper function where you pass the multiplier and it returns you the (ap+target_ap)*multiplier

Reply:
check newer commit, ur commenting on old code still?

Reply:
Looks good now yeah